catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/RoleRepository.java

```diff
@@ -187,6 +191,34 @@ public void storeRelationships(Role role) {
             Relationship.CONTAINS.ordinal());
   }
 
+  public ResultList<Role> getDefaultRolesResultList(Fields fields)
+      throws GeneralSecurityException, UnsupportedEncodingException {
+    List<Role> roles = getDefaultRoles(fields);
+    return new ResultList<>(roles, null, null, roles.size());
+  }
+
+  private List<Role> getDefaultRoles(Fields fields) {
+    List<Role> roles =
+        daoCollection.roleDAO().getDefaultRoles().stream()
+            .map(
+                json -> {
+                  try {
+                    return setFields(JsonUtils.readValue(json, Role.class), fields);
```

Comment:
This throws exception if the json is corrupted or if DB calls to gather data to setFields throws exceptions